python/tank/platform/engine.py

```diff
@@ -1504,14 +1568,15 @@ def _create_dialog_with_widget(self, title, bundle, widget_class, *args, **kwarg
             
         Additional parameters specified will be passed through to the widget_class constructor.
         """
+
         # get the parent for the dialog:
         parent = self._get_dialog_parent()
         
         # create the widget:
         widget = self._create_widget(widget_class, *args, **kwargs)
-        
+
         # apply style sheet
-        self._apply_external_styleshet(bundle, widget)        
```

Comment:
Found this typo! Think we should add a quick backward compatibility line in here for clients that may be using it in subclasses?

`self._apply_external_styleshet = self._apply_external_stylesheet`

or is that silly?